Assets/Scripts/DialogueManager.cs

```diff
@@ -0,0 +1,292 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+[System.Serializable]
+public class SpeakerContainer
+{
+    public string id;
+    public GameObject speakerObject;
+    public string npcVoicePath;
```

Comment:
if you put `[FMODUnity.EventRef]` above line 11, it'll allow me to use the FMOD event browser directly in the script so I won't need to go through the menus. I can do that once this is merged but just so you know!

Reply:
Cool, I just added it!